src/api/SessionAPI.ts

```diff
@@ -0,0 +1,14 @@
+import * as APIUtils from "./APIUtils";
+import { Session } from "../types";
+
+export type ClassIndex = {
+  id: number;
+  name: string;
+  facilitator_id: number;
+};
```

Comment:
can we have this pick off the `ClassInfo` type, since the types should always match up?

Reply:
true, forgot about that. thanks!